ihatemoney/utils.py

```diff
@@ -7,9 +7,11 @@
 import operator
 import os
 import re
+from smtplib import SMTPRecipientsRefused
```

Comment:
Why using this exception rather than the more generic [`SMTPException`](https://docs.python.org/3/library/smtplib.html#smtplib.SMTPException)?

Reply:
Good question, this is code that I just moved around.

Looking at it, `SMTPException` is a subclass of `OSError`, and `socket.error` is actually equal to `OSError`.  Since I already check for `socket.error` it's redundant anyway...

I will look at what happens with older versions of Python (to ease backporting) and fix this mess, thanks for the review.